playbooks/system-criticality.md

```diff
@@ -0,0 +1,50 @@
+---
+title: System Criticality
+description: A framework for treating systems differently according to how critical they are
+---
+
+We'd like to identify gaps in the health and resilience of our platform as well as prioritize efforts to address
+them, but our platform comprises dozens of systems with very different needs and uses. This framework aims to
+recognize those differences with a shared vocabulary, so we can set expectations for different systems accordingly.
+
+## System Criticality Framework
+
+### Level 3 (_critical_):
+
+Systems that are essential to basic business operations such as registration, authentication, browsing, inquiring,
+bidding, and buying. These systems understandably experience a relatively high throughput, and any disruptions can
+have sizable financial and brand impact.
+
+### Level 2 (_important_):
+
+Systems with limited throughput or public-facing functions. Disruptions may interfere with certain business
+operations or have mild financial or brand impact.
+
+### Level 1 (_supported_):
+
+Internal utilities or systems with only occasional usage. Experimentation should be cheap and easy, so we embrace
+that some tools serve only a few individuals or use cases.
```

Comment:
Good question. This was inspired by the conversation on the RFC (https://github.com/artsy/README/issues/176#issuecomment-478702344). If I had to try to explain the difference, it would be that product managers should never need to consider level 0 systems, whereas level 1 systems would appear in their backlogs.

Reply:
How would you modify this (or the level 0 description below) to make that more clear?

Reply:
The linked PR in Potential offers the types of examples I’d look for, so even if there is no 0 yet I think that long term it will provide the full list of examples I’d be looking for.

Nonetheless, I think your added differentiation helps, so adding that to 0 could help.

Reply:
Updated 👍